snapshots/wire-1.4.yaml

```diff
@@ -0,0 +1,12 @@
+# DO NOT MODIFY THIS FILE. See README.md to learn why.
+
+resolver: https://raw.githubusercontent.com/wireapp/wire-server/develop/snapshots/wire-1.3.yaml
+name: wire-1.4
+
+packages:
+- archive: https://github.com/wireapp/http-client/archive/wire-2019-11-04.tar.gz
```

Comment:
Please add a comment to this line describing which commit this archive refers to, and what this includes (e.g. taken from your PR description)

Reply:
Done.